squirrels/_api_routes/base.py

```
@@ -40,35 +38,37 @@ def __init__(self, get_bearer_token: HTTPBearer, project: SquirrelsProject, no_c
     
         # Authorization dependency for current user
         def get_token_from_session(request: Request) -> str | None:
+            access_token = request.session.get("access_token")
+            if access_token is None: return None
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)